s3_upload/s3_upload.py

```diff
@@ -221,6 +221,7 @@ def monitor_directories_for_upload(config, dry_run):
             monitor_dir_config.get("monitored_directories"),
             log_dir=log_dir,
             sample_pattern=monitor_dir_config.get("sample_regex"),
+            max_age=config.get("max_age", 72),
```

Comment:
- `not None` - does not needed handling as we have a default value of 72 in the `.get()`
- `not isinstance(max_age, int)` - not needed as we check type in `utils.validate_config`
- not going to comment this because the code is readable and that isn't needed
<!-- Reviewable comment -OCX_zKt4L48ZdcLibLK:b-8rlx4r -->
<!-- Sent from Reviewable.io -->